plugins/woocommerce-admin/client/core-profiler/style.scss

```diff
@@ -122,3 +124,123 @@
 		}
 	}
 }
+.woocommerce-profiler-business-location {
+	display: flex;
+	flex-direction: column;
+	.woocommerce-profiler-business-location__content {
+		max-width: 550px;
+		width: 100%;
+		display: flex;
+		align-self: center;
+
+		& > div {
+			width: 100%;
+		}
+
+		.woocommerce-profiler-heading {
+			margin-top: 72px;
+			@include breakpoint( '<782px' ) {
+				margin-top: 52px;
+				margin-bottom: 40px;
+				h1,
+				h2 {
+					text-align: left;
+				}
+			}
+		}
+		.woocommerce-profiler-go-to-mystore__button-container {
+			width: 100%;
+			max-width: 400px;
+			@include breakpoint( '<782px' ) {
+				position: absolute;
+				bottom: 20px;
+				padding: 0 20px;
+			}
+		}
+
+		.woocommerce-profiler-go-to-mystore__button {
+			display: flex;
+			width: 100%;
+			justify-content: center;
+			padding: 10px 16px;
+			height: 48px;
+			font-size: 14px;
+			font-weight: 500;
+		}
+		.components-base-control__field {
+			height: 40px;
+		}
+		.woocommerce-profiler-select-control__country {
+			max-width: 400px;
+			margin: auto auto 32px auto;
+			.woocommerce-select-control__option {
+				font-size: 13px;
+			}
+			.woocommerce-select-control__control {
+				height: 40px;
+				padding: 12px;
+				label,
+				input {
+					font-size: 13px;
```

Comment:
It seems that the default component color is `#646970`. I think we need to set it to Gutenberg gray 700.


```
/* Gutenberg/Gray 700 */

color: #757575;
```
![Screenshot 2023-05-10 at 11 24 05](https://github.com/woocommerce/woocommerce/assets/4344253/3fac4157-6026-4932-94c9-5bd416d49bbe)